### PR TITLE
Add ecommerce run/debug settings for PyCharm.

### PR DESCRIPTION
**Pull request by robrap**
FYI: @jmbowman

### CODE REVIEW

File: docs/pycharm_integration.rst [hunk 1]
```diff
@@ -93,9 +93,9 @@ PyCharm -> Preferences -> Languages & Frameworks -> Django
 
 Here are some example settings you might use:
 
-Django Project Root: /Path/to/docker_devstack/ecommerce
-Settings: ecommerce/settings/devstack.py
-Manage Script: manage.py
+Django Project Root: /Path/to/docker_devstack
+Settings: ecommerce/ecommerce/settings/devstack.py
+Manage Script: ecommerce/manage.py
```
Comment (staubina) [inline docs/pycharm_integration.rst]: The 3 lines above here differ from what my setup required.  Could this be based somehow on where you start the Project?

**Review by staubina — CHANGES_REQUESTED** [review]
The Helper settings differ from mine. I am worried about confusion. 

Otherwise the updates look good.

File: docs/pycharm_integration.rst (hunk 1 above)
Comment (robrap) [reply docs/pycharm_integration.rst]: I'll provide both options and explain why.

**Comment by robrap** [comment]
@staubina: I made the changes.  Let me know what you think.

**Review by staubina — APPROVED** [review]

**Review by BenjiLee — APPROVED** [review]

**Pull request merged**